RadixWallet/Features/ProfileBackupsFeature/RestoreProfileFromBackup/Children/SelectBackup/SelectBackup+Reducer.swift

```diff
@@ -111,22 +116,13 @@ public struct SelectBackup: Sendable, FeatureReducer {
 			state.destination = .recoverWalletWithoutProfileCoordinator(.init())
 			return .none
 
-		case let .selectedProfileHeader(header):
-			state.selectedProfileHeader = header
+		case let .selectedProfile(profile):
+			state.selectedProfile = profile
 			return .none
 
-		case let .tappedUseCloudBackup(profileHeader):
-			return .run { send in
-				let (snapshot, containsP2PLinks) = try await backupsClient.lookupProfileSnapshotByHeader(profileHeader)
-				guard let snapshot else {
-					await send(.internal(.snapshotWithHeaderNotFoundInCloud(profileHeader)))
-					return
-				}
-				await send(.delegate(.selectedProfileSnapshot(snapshot, isInCloud: true, containsP2PLinks: containsP2PLinks)))
-			} catch: { error, send in
-				loggerGlobal.error("Failed to load profile snapshot with header, error: \(error), header: \(profileHeader)")
-				await send(.internal(.snapshotWithHeaderNotFoundInCloud(profileHeader)))
-			}
+		case let .tappedUseCloudBackup(profile):
+			// TODO: pass p2pLinks
+			return .send(.delegate(.selectedProfile(profile, isInCloud: true, containsP2PLinks: false)))
```

Comment:
So if `cloudBackupClient.loadAllProfiles()` returns `containsP2PLinks` for every profile, this action will also pass the `containsP2PLinks` flag.